content/cucumber/step-definitions.md

```diff
@@ -253,3 +253,60 @@ Given("I have {int} {color} balls", function (int1, color) {
 });
 ```
 {{% /block %}}
+
+# How do I group my step definitions?
+
+Avoid writing similar step definitions which can lead to clutter. While documenting your steps helps, making use of **helper methods** to abstract them can do wonders.
+
+For eg, if you have the following steps :
+
+```
+    Given I go to the home page
+    Given I check the about page of the website 
+    Given I get the contact details
+```
+
+and all of these steps visit the **Home**, **About** and **Contact** pages, you might be writing _redundant steps_. While the underlying code for these steps could be different, their **behaviour** is essentially the same, ie.
```

Comment:
You can use one `*` if you want text to be *italic*, and two `**` if you want text to be **bold**. While underscores may do same / similar, this will help with consistency.